src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -266,6 +267,13 @@ public void respawn()
 		alive = true;
 	}
 
+	public void moveInstantly(Location moveLocation) {
+		PlayerMoveEvent event = new PlayerMoveEvent(this, this.getLocation(), moveLocation);
+		Bukkit.getPluginManager().callEvent(event);
+		if(event.isCancelled() == false)
```

Comment:
Comparing a boolean to a boolean to get a boolean is redundant.

```suggestion
		if (!event.isCancelled())
```